software/raft/raft_server_riscv/firesim_riscv.cmake

```diff
@@ -0,0 +1,13 @@
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR riscv)
+set(tools /home/centos/chipyard/riscv-tools-install)
```

Comment:
again, this assumes the repo is in `$HOME`

Reply:
This entire file actually isn't needed any more, so I just took it out.